### ISSUE TITLE
Week name

### ISSUE THREAD

**Issue by miladmodaresi** [jalaali/moment-jalaali#196]
Hello 
how can I get the week name?
for example:
result of `moment(ts).format('dddd')` in the moment js is **Sunday**
but I want when I call that code execute
**یکشنبه**

**Comment by behrang** [comment]
Load Persian module before that: `moment.loadPersian()`.

**Comment by shafaprince** [comment]
How to use Miladi and Shamsi the same time? ( I need both sunday and یکشنبه)
like `m.format('dddd') ` // sunday
`m.format('jDDDD')` // یکشنبه

**Comment by behrang** [comment]
m.format('jDDDD dddd')

**Comment by behrang** [comment]
No, sorry, my previous comment is wrong. There is no easy way for what you need, since `loadPersian` changes the language globally.
